### PR TITLE
Manage bookmarks

### PR DESCRIPTION
**Pull request by apurvag123**

### CODE REVIEW

**Comment by apurvag123** [comment]
#126

File: src/iOS/Xamarin.iOS/Samples/Map/ManageBookmarks/ManageBookmarks.cs [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+{
+    [Register("ManageBookmarks")]
+    public class ManageBookmarks : UIViewController
+    {
+        // Create and hold reference to the used MapView
+        private MapView _myMapView = new MapView();
+
+        public ManageBookmarks()
+        {
+            this.Title = "Manage bookmarks";
```
Comment (anttikajanus) [inline src/iOS/Xamarin.iOS/Samples/Map/ManageBookmarks/ManageBookmarks.cs]: Remove this.

File: src/iOS/Xamarin.iOS/Samples/Map/ManageBookmarks/ManageBookmarks.cs [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+            _myMapView.Map.Bookmarks.Add(myBookmark);
+        }
+
+        private void OnAddBookmarksButtonClicked(object sender, EventArgs e)
+        {
+            //Create Alert for bookmark name
+            var textInputAlertController = UIAlertController.Create("Provide the bookmark name", string.Empty, UIAlertControllerStyle.Alert);
+            
+            //Add Text Input
+            textInputAlertController.AddTextField(textField => {
```
Comment (anttikajanus) [inline src/iOS/Xamarin.iOS/Samples/Map/ManageBookmarks/ManageBookmarks.cs]: What does this do?

**Comment by anttikajanus** [comment]
Good one. Please have a look on the 2 comments and merge when ready.

File: src/iOS/Xamarin.iOS/Samples/Map/ManageBookmarks/ManageBookmarks.cs (hunk 2 above)
Comment (apurvag123) [inline src/iOS/Xamarin.iOS/Samples/Map/ManageBookmarks/ManageBookmarks.cs]: Adding text field to textInputAlertController needs an action parameter. It cannot be null

File: src/iOS/Xamarin.iOS/Samples/Map/ManageBookmarks/ManageBookmarks.cs (hunk 1 above)
Comment (apurvag123) [inline src/iOS/Xamarin.iOS/Samples/Map/ManageBookmarks/ManageBookmarks.cs]: Remove title?

**Pull request merged**